commands/initial/templates/e2e/example.e2e-spec.ts

```diff
@@ -0,0 +1,10 @@
+import {examplePo} from './page-objects/example.po';
+
+describe('Example Page', () => {
+
+    beforeEach(() => examplePo.go());
+
+    it('should welcome user to the example page', () => {
+        expect(examplePo.welcome.getText()).toBe('Welcome');
```

Comment:
I'm not generally a fan of default tests. I'd even like to get rid of the ones added to scaffolded unit tests but they're good sanity checks.

For this, it seems that anytime a library is generated, the dev has to remove this test--is that true? If it is, can we remove it and keep the rest of the setup?